_source/logzio_collections/_log-sources/kubernetes.md

```diff
@@ -65,52 +73,101 @@ The logzio-k8s image comes pre-configured for Fluentd to gather all logs from th
     see [log shipping troubleshooting]({{site.baseurl}}/user-guide/log-shipping/log-shipping-troubleshooting.html).
 {:.tasklist.firstline-headline}
 
-## Disabling systemd input
+</div>
+<!-- tab:end -->
 
-To suppress Fluentd system messages, set the `FLUENTD_SYSTEMD_CONF` environment variable to `disable` in your Kubernetes environment.
 
-<!-- ## Customizing the configuration {#customizing-the-configuration}
+<!-- tab:start -->
+<div id="custom-config">
+
+## Deploy logzio-k8s with custom configuration
+
+You can customize the configuration of the Fluentd container.
+This is done using a ConfigMap that overwrites the default DaemonSet.
+
+###### To deploy logzio-k8s
+
+1.  Store your Logz.io credentials
+
+    Save your Logz.io shipping credentials as a Kubernetes secret.
+
+    {% include log-shipping/replace-vars.html token=true listener=true %}
```

Comment:
Maybe we should be more detailed with the listener example.
Instead of: `region’s listener host (for example, listener.logz.io)...`
->`region’s listener host (for example, if your logz.io account is on `N. Virginia region` use listener.logz.io)...`

Reply:
Let's keep this as is unless there's a good reason to change. This is docs-wide and across quite a few github repos at this point.